### ISSUE TITLE
Pls explain 'cfg_parallel'?

### ISSUE THREAD

**Issue by chen-yy20** [NUS-HPC-AI-Lab/VideoSys#191]
Hello! Thank you for your concise implementation of VideoSys. I appreciate your work, but I have a small question regarding a specific part of the code. In the following snippet:
```
if enable_cp and sp_size % 2 == 0:
    sp_size = sp_size // 2
    cp_size = 2
else:
    cp_size = 1
```
I'm curious about the purpose of `cfg_parallel`and why it needs to be set to 2. Could you please elaborate on this?

**Comment by oahzxl** [comment]
in sampling, we will use classifer free guidence (cfg), which will add another batch to original data. so we will have a batch size of 2. cfg parallel is to do data parallel for this dimension.

**Comment by chen-yy20** [comment]
Understood, thanks!

**Comment by chen-yy20** [comment]
Hi, I got some new observation. I ran OpenSora inference example with different configuration, and  setting `enable_cp` to be `False` can always get speedup. Compared with SP, CP neither reduces memory usage nor increases computation speed. Why is it set as the first priority?

**Comment by oahzxl** [comment]
yeah indeed. thanks for your feedback. it will be slower with cp parallel. disable it now!

**Comment by oahzxl** [comment]
close due to inactive
